### PR TITLE
Reconfigurable courtyard

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/DEV-254

### CODE REVIEW

File: placeSpecificContent/hq/courtyard/entityResources/podium/buttons/courtyardPodiumButton_server.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+
+(function() {
+
+    var BUTTON_PRESS_OFFSET = 0.02;
+    var DEFAULT_LOCAL_Y_POSITION_M = 0.4771347;
+    var RED = "https://hifi-content.s3.amazonaws.com/jimi/environment/201802_Shop/buttons/buttonRed.fbx";
```
Comment (MiladNazeri) [inline placeSpecificContent/hq/courtyard/entityResources/podium/buttons/courtyardPodiumButton_server.js]: Same CDN question

File: placeSpecificContent/hq/courtyard/entityResources/podium/courtyardPodium_server.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    };
+
+    Gate.prototype = {
+        remotelyCallable: ['openGate', 'closeGate'],
+
+        // Sets up this entity's closed and opened positions
+        preload: function(entityID){
+            _this.entityID = entityID;
+            var name = Entities.getEntityProperties(_this.entityID, 'name').name;
+            if (name === "Bingo Scanner Entry Gate") {
```
Comment (MiladNazeri) [inline placeSpecificContent/hq/courtyard/entityResources/podium/courtyardPodium_server.js]: I think these Bingo things are left over?

File: placeSpecificContent/hq/courtyard/entityResources/podium/courtyardPodium_server.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                            localPosition: openedLocalPosition,
+                            velocity: {x: 0, y: 0, z: 0}
+                        }
+                    );
+                    Script.clearInterval(gatePositionCheckInterval);
+                    gatePositionCheckInterval = false;
+                }
+            }, POSITION_CHECK_INTERVAL_MS);
+        },
+
```
Comment (MiladNazeri) [inline placeSpecificContent/hq/courtyard/entityResources/podium/courtyardPodium_server.js]: This file might need to be looked at, I think there are a lot of Bingo left overs.

File: placeSpecificContent/hq/courtyard/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Reconfigurable Courtyard
+
+This allows the courtyard setup to be chnged via buttons. There are 3 preset types: open, stage, and table.
+
+
+
+## Releases
+### v1.0 | []()
```
[the remark this answers is not in the document]
Comment (RebeccaStankus) [reply placeSpecificContent/hq/courtyard/README.md]: no, this isn't the final 1st version until I get models and sounds and correct IDs for the hard coded entities. I added the DO NOT MERGE for now.

File: placeSpecificContent/hq/courtyard/entityResources/podium/courtyardPodium_server.js (hunk 2 above)
Comment (RebeccaStankus) [reply placeSpecificContent/hq/courtyard/entityResources/podium/courtyardPodium_server.js]: Sorry, this was not supposed to be included. I ended up not having a server file on the podium.

**Pull request closed without merging**